Sources/NIO/Linux.swift

```diff
@@ -103,9 +103,9 @@ internal enum Linux {
     static let SOCK_NONBLOCK = Int32(bitPattern: Glibc.SOCK_NONBLOCK.rawValue)
```

Comment:
please also remove all the changes in this file, I think Xcode's 'rename' function got properly confused here...